Yoast/YoastSEO.js
# Add possessive inflection to the morphological analysis

## Summary

This PR can be summarized in the following changelog entry:

* Adds analysis of possessive nouns (e.g., "boy's" in "the boy's car") so that noun forms can be correctly built. 
* Prohibits building of verb forms from possessive forms (to prevent cases like "boy'sing").

## Relevant technical choices:

* This PR also takes care of normalized/non-normalized apostrophes to be correctly recognized in the text.
* More relevant specs were added

## Test instructions

This PR can be tested by following these steps:

* Use browserified to check titleKeyword assessment.
* Try using different forms of nouns (singular, plural, singular possessive, plural possessive) in the keyword field and in the title. Check if the assessment returns correct result.
* Check if normalized and non-normalized apostrophes are parsed correctly.

Fixes #1535 Fixes #1514
